pkg/reconciler/githubsource/githubsource.go

```diff
@@ -183,7 +193,20 @@ func (r *reconciler) reconcile(ctx context.Context, source *sourcesv1alpha1.GitH
 			}
 			source.Status.WebhookIDKey = hookID
 		}
+	} else {
+		return nil
 	}
+
+	// Only create EventTypes for Broker sinks.
+	if source.Spec.Sink.Kind == "Broker" {
```

Comment:
why?

Reply:
This is what we said in the proposal. Registry is mainly for Broker/Trigger model, at least for the MVP. If you are just sending stuff to another sink, then the EventType won't be added to the registry. 

There may be a more "typed" way to do this check?

Reply:
Why filter?

What if the source goes `source -> channel -> broker`?

Reply:
Our current EventType CRD has a mandatory `broker` field. 
I guess we should have called it `sink` if we were aiming for other cases.
I'm somewhat reluctant to change it right now, as that is what the community accepted. 
But I'm open to discussions. What do other people think @vaikas-google @grantr @Harwayne

Reply:
I'd prefer to leave this as is for now. I could see a use case where you could list event types that are wired for a given channel though. That's what the agreed upon design was so we can revisit at a later date.

Reply:
Agree. Will leave it as is for now, and then we can iterate over it.